### PR TITLE
Added test for AddPlayers and RemovePlayers in TeamService

### LINKED ISSUE #353: Add test to add/remove players from team
Parent #298

### PR DESCRIPTION
**Pull request by ViznyakS**
### Summary

Add test for AddPlayers and RemovePlayers in TeamService

closes #353

#### Design

- [ ] Web API layer contains as little logic as possible
- [ ] Domain objects use semantic names

#### Perfromance

- [ ] In case of any DB query was changed: attach generated SQL for review
- [ ] Number of DB roundtrips should be as low as possible

#### Unit Tests

- [ ] Naming: Initial state and expected result are properly stated
- [ ] Test follows [AAA](https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80) pattern.
- [ ] Test is Isolated
- [ ] Mock instances and expected instances do not share references
- [ ] When new property is added all related comparers are updated

### CODE REVIEW

**Comment by ViznyakS** [comment]
I thought that we don't need check domain entity behavior, but was not sure) Delete this current tests(not all)? And write test t o check the call queries and authorization ?

File: tests/VolleyManagement.UnitTests/Services/TeamService/TeamServiceTests.cs [hunk 1]
```diff
@@ -893,6 +893,125 @@ public void ChangeCaptain_PlayerIsPlayerOfAnotherTeam_ValidateExceptionThrown()
         }
         #endregion
 
+        #region Add and remove players
+        /// <summary>
+        /// Test for AddPlayers() method. 
+        /// </summary>
+        [TestMethod]
+        public void AddPalyers_ToTeam_AllPlayerNew_TeamUpdated()
```
Comment (sdiachen) [inline tests/VolleyManagement.UnitTests/Services/TeamService/TeamServiceTests.cs]: 1) Typo
2) Test name should consist of 3 parts
3) What does it mean "AllPlayerNew"?

File: tests/VolleyManagement.UnitTests/Services/TeamService/TeamServiceTests.cs [hunk 2]
```diff
@@ -893,6 +893,125 @@ public void ChangeCaptain_PlayerIsPlayerOfAnotherTeam_ValidateExceptionThrown()
...
+            VerifyEditTeam(testTeam, Times.Once());
+            TestHelper.Equals(exeptedTeam, testTeam);
+        }
+
+        /// <summary>
+        /// Test for AddPlayers() method. 
+        /// Should throw MissingEntityException
+        /// </summary>
+        [TestMethod]
+        public void AddPalyers_ToTeam_MissingEntityExceptionThrown()
```
Comment (sdiachen) [inline tests/VolleyManagement.UnitTests/Services/TeamService/TeamServiceTests.cs]: There is no precondition part in the name

File: tests/VolleyManagement.UnitTests/Services/TeamService/TeamServiceTests.cs [hunk 3]
```diff
@@ -893,6 +893,126 @@ public void ChangeCaptain_PlayerIsPlayerOfAnotherTeam_ValidateExceptionThrown()
         }
         #endregion
 
+        #region Add and remove players
+
+        /// <summary>
+        /// Test for AddPlayers() method. 
+        /// </summary>
+        [TestMethod]
+        public void AddPlayers_AllPlayersAddedToTeam_TeamUpdated()
```
Comment (sdiachen) [inline tests/VolleyManagement.UnitTests/Services/TeamService/TeamServiceTests.cs]: AllPlayersAddedToTeam sounds like expected result not a precondition

**Pull request merged**